src/tests/utils/common.ts

```diff
@@ -91,3 +91,11 @@ export const filterArrayBySearchText = (
 		})
 	})
 }
+
+export const getRecordsWithPagination = (
+	array: any[],
+	offset: number,
+	limit: number,
+) => {
+	return array.slice(offset, offset + limit + 1)
```

Comment:
should extract offset + limit + 1  to const for better self documentation
const end = offset + limit + 1